### PR TITLE
Add raw data backup and restore

### PR DESCRIPTION
**Pull request by btat**
Add chapter and https://github.com/SUSE/doc-cap/commit/2e5d85949930d31922322a570f2232f6f0ce381f can now refer to something that exists

### CODE REVIEW

**Comment by cjschroder** [comment]
Nice, this looks good. I have some questions-- see inline review comments.

File: xml/cap_admin_backup-restore.xml [hunk 1]
```diff
@@ -521,5 +523,249 @@
...
+    <step>
+     <para>
+      Connect to the <literal>blobstore</literal> pod:
+     </para>
+<screen>&prompt.user;kubectl exec -it blobstore-0 --namespace scf -- env /bin/bash</screen>
+    </step>
+    <step>
+     <para>
+      Create an archive of the blobstore directory to preserve all needed files (droplets and other artifacts) then disconnect from the pod:
+     </para>
```
Comment (cjschroder) [inline xml/cap_admin_backup-restore.xml]: Should say what the other artifacts are, or xref to a list

File: xml/cap_admin_backup-restore.xml [hunk 2]
```diff
@@ -521,5 +523,249 @@
...
+  ccdb' > <replaceable>/tmp/ccdb-src.sql</replaceable>
+</screen>
+    </step>
+    <step xml:id="get-encryption-key">
+     <para>
+      Next, obtain the CCDB encryption key(s). The method used to capture the key will depend on whether <literal>current_key_label</literal> has been defined on the source cluster. This value is defined in <filename>/var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml</filename> of the <literal>api-group-0</literal> pod and also found in various tables of the MySQL database.
+     </para>
+     <para>
+      Begin by examining the config file for the<literal>current_key_label</literal> setting:
+     </para>
```
Comment (cjschroder) [inline xml/cap_admin_backup-restore.xml]: configuration, please, not config.

File: xml/cap_admin_backup-restore.xml [hunk 3]
```diff
@@ -521,5 +523,249 @@
...
+  <sect2 xml:id="sec.cap.raw-data-restore-procedure">
+   <title>Performing a raw data restore</title>
+   <para>
+    Perform the following steps to restore your backed up data to the target SCF deployment.
+   </para>
+   <important>
+    <title>Ensure access to the correct SCF deployment</title>
+     <para>
+      Working with multiple &kube; clusters simultaneously can be confusing. Ensure you're communicating with the desired cluster <link xlink:href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable">by setting <literal>$KUBECONFIG</literal> correctly</link>.
+     </para>
```
Comment (cjschroder) [inline xml/cap_admin_backup-restore.xml]: you are, not you're

File: xml/cap_admin_backup-restore.xml [hunk 4]
```diff
@@ -521,5 +523,249 @@
...
+  CC_DB_ENCRYPTION_KEYS:
+  <replaceable>migrated_key</replaceable>: "<replaceable>OLD_CLUSTER_DB_ENCRYPTION_KEY</replaceable>"
+</screen>
+      </listitem>
+     </itemizedlist>
+    </step>
+    <step>
+     <para>
+       Deploy a non-high-availability configuration of SCF (see <xref linkend="sec.cap.install-scf-prod"/>) and waiting until all pods are ready before proceeding.
+     </para>
```
Comment (cjschroder) [inline xml/cap_admin_backup-restore.xml]: wait, not waiting

File: xml/cap_admin_backup-restore.xml [hunk 5]
```diff
@@ -521,5 +523,249 @@
...
+        Run the rotation for the encryption keys:
+       </para>
+       <screen>&prompt.user;kubectl exec --namespace scf api-group-0 -- bash -c 'export PATH=/var/vcap/packages/ruby-2.4/bin:$PATH ; export CLOUD_CONTROLLER_NG_CONFIG=/var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml ; cd /var/vcap/packages/cloud_controller_ng/cloud_controller_ng ; /var/vcap/packages/ruby-2.4/bin/bundle exec rake rotate_cc_database_key:perform'</screen>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>
+      The data restore is now complete. Run commands such as <command>cf apps</command>, <command>cf marketplace</command>, <command>cf services</command>, etc. and verify data from the old cluster is returned.
+     </para>
```
Comment (cjschroder) [inline xml/cap_admin_backup-restore.xml]: Don't use "etc." We do not handwave, we provide specifics. "Run some cf commands, such as cf apps, cf marketplace, or cf services, and verify data from the old cluster is returned."

File: xml/cap_admin_backup-restore.xml (hunk 3 above)
Comment (btat) [reply xml/cap_admin_backup-restore.xml]: Addressed in https://github.com/SUSE/doc-cap/pull/280/commits/fcde67d3da1a19b5743da5d0fb77df36f5e37f01

File: xml/cap_admin_backup-restore.xml (hunk 5 above)
Comment (btat) [reply xml/cap_admin_backup-restore.xml]: Addressed in https://github.com/SUSE/doc-cap/pull/280/commits/fcde67d3da1a19b5743da5d0fb77df36f5e37f01

File: xml/cap_admin_backup-restore.xml (hunk 4 above)
Comment (btat) [reply xml/cap_admin_backup-restore.xml]: Addressed in https://github.com/SUSE/doc-cap/pull/280/commits/fcde67d3da1a19b5743da5d0fb77df36f5e37f01

**Comment by cjschroder** [comment]
This is good, nice work. :+1:

**Pull request merged**